Source/WebGPU/WGSL/AST/ASTVisitor.h

```diff
@@ -37,53 +37,53 @@ class Visitor {
     virtual ~Visitor() = default;
 
     // Shader Module
-    virtual void visit(AST::ShaderModule&);
-    virtual void visit(AST::GlobalDirective&);
```

Comment:
Normally I'd recommend doing this stuff in a different patch, but I guess it's fine here since we're trying to get off the ground.